integration-tests/src/multichain/mod.rs

```diff
@@ -71,19 +72,36 @@ impl Nodes<'_> {
         }
     }
 
-    pub async fn add_node(
+    pub fn near_acc_sks(&self) -> HashMap<AccountId, SecretKey> {
+        let mut account_to_urls = HashMap::new();
```

Comment:
hmm, why is `account_to_urls`? Shouldn't it be `account_to_sk`?

Reply:
typo